src/app/modules/main/communities/module.nim

```diff
@@ -198,54 +198,61 @@ proc createMemberItem(self: Module, memberId, requestId: string, status: Members
     membershipRequestState = status,
   )
 
-method getCommunityItem(self: Module, c: CommunityDto): SectionItem =
-  # TODO: unite bannedMembers, pendingMemberRequests and declinedMemberRequests
-  var members: seq[MemberItem] = @[]
-  for member in c.members:
-    if c.pendingAndBannedMembers.hasKey(member.id):
-      let communityMemberState = c.pendingAndBannedMembers[member.id]
-      members.add(self.createMemberItem(member.id, "", toMembershipRequestState(communityMemberState)))
+method getCommunityItem(self: Module, community: CommunityDto): SectionItem =
```

Comment:
in `CommunityDto` members are divided into categories in separate containers and here we merge them into single sequence. Could we go further with simplification/unification and keep also only one list there?

Reply:
I would keep the DTOs the same as the status-go object representation. We have already multiple places where the DTO is not a match to the status-go object and it causes a lot of issues of expected properties not being there in some cases.

I think it's fine to merge them in a model, because that's just what we expose to the view, but the DTO makes sense to stay as a faithful representation of the backend.

I guess the way to refactor it would be to change status-go to merge the member lists, but I think that would be very complex and prone to bugs, because we make a lot of assumptions that users in the `member` list are the ones that are joined.